opendocdb-cts/main.go

```diff
@@ -114,12 +114,12 @@ func runCommand(ctx context.Context, l *slog.Logger) error {
 		return err
 	}
 
-	if err = r.Setup(ctx, f); err != nil {
-		return err
-	}
```

Comment:
Why is this change necessary? I'm lacking the context.

Reply:
https://github.com/OpenDocDB/cts/blob/8c04642a56beb17646814693ee6ac469b5df1033/opendocdb-cts/internal/data/testsuite.go#L37-L44

To make test suites independent, they should not rely on changes made by the previous random test suite. It only worked with test cases that did not modify data, but they do in FerretDB.